sdk/monitor/opentelemetry-exporter-azure-monitor/src/export/trace.ts

```diff
@@ -140,4 +149,21 @@ export class AzureMonitorTraceExporter implements SpanExporter {
       this._logger.warn(`Failed to fetch persisted file`, err);
     }
   }
+
+  private _isNetworkError(error: Error): boolean {
+    if (error instanceof RestError) {
+      if (
+        error &&
+        error.code &&
+        (error.code === "ETIMEDOUT" ||
```

Comment:
These all seems sensible to me but I'm wondering how you curated this list?

Reply:
Same checks as systemErrorRetryPolicy  https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-https/src/policies/systemErrorRetryPolicy.ts#L153